charts/zwavejs2mqtt/0.0.1/test_values.yaml

```diff
@@ -113,6 +113,8 @@ additionalVolumes: []
 appVolumesEnabled: false
 emptyDirVolumes: true
 
+appAdditionalServicesEnabled: false
+
```

Comment:
additional services need to be included in the test... Ingress does not however.

Reply:
I don't think this should be required. Or what do you do when there is not an additional service to offer?

The main service used for health checks and such is still defined under the original service heading.

Reply:
I  just noticed I had the web-socket service enabled by default. That is my mistake. It should be set `enabled=false`

This not something you just open and port for... It must also be enabled in the application settings, after installation.

Reply:
I dont think there is a risk with adding it and doing so makes sure extra service creation is checked.

Reply:
these still nothing running on that port until the web-socket is enable in the application itself

Primary use by the developer is using MQTT, which does not require an additional ports.

The web-socket is optional (new feature) for connecting to the new Home Assistant integration, without MQTT in the middle.

I imagine existing users will continue using MQTT and never enable the web-socket